app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, source_name, destination_name = "", "", ""
+    try:
+        cmd, source_name, destination_name = command.split()
+    except Exception:
+        print("Error in cmd!")
+    print(cmd, source_name, destination_name)
+
+    dir_str = ""
+
+    if cmd == "mv":
+        if "/" in destination_name:
+            list_dir = destination_name.split("/")
+
+            for i in range(len(list_dir) - 1):
+                dir_str += list_dir[i] + "/"
+
+                try:
+                    print(dir_str)
+                    os.mkdir(dir_str)
+                except Exception:
+                    print("Err_dir")
```

Comment:
Use os.makedirs with exist_ok to avoid errors.
```suggestion
                os.makedirs(dir_str, exist_ok=True)
```